packages/bpk-component-infinite-scroll/stories.js

```diff
@@ -172,7 +173,7 @@ storiesOf('bpk-component-infinite-scroll', module)
           onClick={() => {
             const newElements = [];
             const k = Math.floor(Math.random() * 10);
-            for (let i = 0; i < 100; i = 1) {
+            for (let i = 0; i < 100; i += 1) {
```

Comment:
Is this a best practise or does `i++` work/look better?

Reply:
eslint discourages using unary operators

Reply:
No problem! In which case 👍